doc-content/enterprise-only/installation/ssh-configuring-proc.adoc

```diff
@@ -0,0 +1,86 @@
+[id='ssh-configuring-proc_{context}']
+
+= Configuring SSH to use RSA
+SSH is used to clone Git repositories. By default, the DSA encryption algorithm is provided by Business Central. However, some SSH clients, for example SSH clients in the Fedora 23 environment, use the RSA algorithm instead of the DSA algorithm. Business Central contains a system property that you can use to switch from DSA to RSA if required. 
+
+[NOTE]
+====
+SSH clients on supported systems, for example Red Hat Enterprise Linux 7, are not affected by this issue.
+====
+
+.Procedure
+
+Complete one of the following tasks to enable this system property:
+
+* Modify the `~/.ssh/config` file on client side as follows to force the SSH client to accept the deprecated DSA algorithm:
++
+[source]
+----
+Host <SERVER_IP>
+       HostKeyAlgorithms +ssh-dss
+----
+ 
+* Include the `-Dorg.uberfire.nio.git.ssh.algorithm=RSA` parameter when you start {CENTRAL}, for example:
++
+[source]
+----
+$ ./standalone.sh -c standalone-full.xml 
+-Dorg.uberfire.nio.git.ssh.algorithm=RSA
+----
+
+= Verifying the {PRODUCT} installation
```

Comment:
In addition to **= Configuring SSH to use RSA**  This is the new chapter in the doc right?

Reply:
Yes